assets/ca/ca.querytranslator.js

```diff
@@ -255,6 +271,19 @@ var caUI = caUI || {};
 	};
 
 	/**
+	 * Determine whether the given list of tokens is all words and whitespace.
+	 * @param {Array} tokens
+	 * @returns {boolean}
+     */
+	isRawSearchText = function (tokens) {
```

Comment:
This is a bit ugly, with `lodash` it's a lot neater, but it works.  Could have also used a plain `for` loop.